### PR TITLE
misc dep bumps

### PR DESCRIPTION
**Pull request by turadg**
no ticket

## Description

Had free hands during SnT

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,9 +24,9 @@
     "@endo/eslint-plugin": "^0.4.4",
     "@endo/ses-ava": "^0.2.40",
     "@jessie.js/eslint-plugin": "^0.4.0",
-    "@typescript-eslint/eslint-plugin": "^7.1.0",
-    "@typescript-eslint/parser": "^7.1.0",
-    "ava": "^5.3.0",
+    "@typescript-eslint/eslint-plugin": "^7.8.0",
+    "@typescript-eslint/parser": "^7.8.0",
+    "ava": "^6.1.3",
```
Comment (samsiegart) [inline package.json:29]: I'd like to try removing ava altogether since we don't have any ava test here anymore afaik

**Review by samsiegart — CHANGES_REQUESTED** [review]
Seems like build is failing now, maybe because of yarn version?

**Comment by turadg** [comment]
> Seems like build is failing now, maybe because of yarn version?

I think it's the TS bump parsing this differently, or this was bumped
```
../../node_modules/@cosmjs/encoding/build/utf8.d.ts(1,1): error TS1490: File appears to be binary.
```

I'll come back to this in another meeting :)

**Review by samsiegart — APPROVED** [review]
Sorry for delay, I was having trouble testing with `yarn link`. I'm just going to approve because the ui tutorial points to a stable version anyway. When this publishes, I'll validate, publish a new minor version, and update the ui tutorial to use it.

**Pull request merged**